src/vs/workbench/contrib/chat/browser/modelPicker/modelPickerActionItem.ts

```diff
@@ -39,84 +35,45 @@ export class ModelPickerActionItem extends ActionViewItem {
 			...action,
 			label: currentModel.metadata.name,
 			tooltip: localize('chat.modelPicker.label', "Pick Model"),
-			run: () => { /* Will be overridden by our click handler */ }
+			run: () => { }
 		};
 
-		super(undefined, actionWithLabel, { label: true });
+		super(actionWithLabel, (container, labelRenderer) => {
```

Comment:
I think overriding a method would be much more natural than this nested callback pasted into super thing